### PR TITLE
OEM: IONOS Cloud Images

### PR DESCRIPTION
**Pull request by tuunit**
# [Title: describe the change in one sentence]

[ describe the change in 1 - 3 paragraphs ]

## How to use

[ describe what reviewers need to do in order to validate this PR ]

## Testing done

[Describe the testing you have done before submitting this PR. Please include both the commands you issued as well as the output you got.]

- [ ] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [ ] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

File: systemd/system/ignition-delete-config.service [hunk 1]
```diff
@@ -10,6 +10,7 @@ ConditionKernelCommandLine=|coreos.oem.id=virtualbox
 ConditionKernelCommandLine=|ignition.platform.id=vmware
 ConditionKernelCommandLine=|flatcar.oem.id=vmware
 ConditionKernelCommandLine=|coreos.oem.id=vmware
+ConditionKernelCommandLine=|flatcar.oem.id=ionoscloud
```
Comment (tormath1) [inline systemd/system/ignition-delete-config.service:13]: I don't see an implementation for `ignition-rmcfg` in the Ignition PR. 
That said, you could easily implement it as Ignition would just need to remove this file: `/var/lib/cloud/seed/nocloud/user-data` once consumed.

Comment (tuunit) [reply systemd/system/ignition-delete-config.service:13]: but we need to keep it for cloud-init

Comment (tormath1) [reply systemd/system/ignition-delete-config.service:13]: In this case we can drop this line.
```suggestion
```

Comment (tuunit) [reply systemd/system/ignition-delete-config.service:13]: This PR has been closed a couple of days ago 😅 So nothing to merge here anyhow

Comment (tuunit) [reply systemd/system/ignition-delete-config.service:13]: Replaced by https://github.com/flatcar/init/pull/127

Comment (tormath1) [reply systemd/system/ignition-delete-config.service:13]: Ah sorry, I clicked on the link from the `scripts` PR description. :D

**Pull request closed without merging**